solution-templates/mongodb-atlas/microservices-api-semantic-search/README.md

```diff
@@ -0,0 +1,123 @@
+# Microservices API with Semantic Search using MongoDB Atlas and Amazon Bedrock
+
+## Overview
+This solution template deploys a complete microservices architecture with semantic search capabilities, leveraging MongoDB Atlas for data storage and vector search, combined with Amazon Bedrock for AI-powered embeddings.
+
+### What This Solution Deploys
+
+This template automates the deployment of:
+
+1. **MongoDB Atlas Cluster** - M0 free tier cluster with vector search capabilities
+2. **MongoDB Data API** - Serverless REST API for database operations
+3. **Semantic Search Service** - Lambda function integrating Bedrock embeddings with MongoDB vector search
+4. **Todos Microservice** - Complete CRUD API for task management
+5. **API Gateway** - Unified REST API endpoint with IAM authentication
+6. **Sample Data** - Pre-loaded travel dataset with vector embeddings
+
+### Architecture
+
+The solution implements a modern microservices pattern:
+- MongoDB Atlas provides the database layer with built-in vector search
+- AWS Lambda functions implement business logic
+- Amazon Bedrock generates embeddings for semantic search
+- API Gateway provides a secure, scalable API layer
+- All components use IAM authentication for security
+
+### Prerequisites
+
+- AWS Account with appropriate permissions
+- MongoDB Atlas account ([Start free trial](https://www.mongodb.com/cloud/atlas/register))
+- Terraform installed (version 1.0+)
+- AWS CLI configured
+- MongoDB Atlas API keys
```

Comment:
It'd be great if this could be linked to some official MongoDB docs as to how to create the key.

Reply:
link provided